### DISCUSSION TITLE
Why can't I install filprofiler?

### DISCUSSION THREAD

**Discussion by RayenHarchay** [pythonspeed/filprofiler#332] [Q&A] [answered]
Hey,

Sorry if this is a bit primitive, but I cannot seem to install this package.

![image](https://user-images.githubusercontent.com/82264889/158163510-2eaea7c7-d333-4feb-aaa8-d33e8a34585e.png)

I am trying to figure out a fix for a memory allocation issue/bug and I was counting on this module to show me the way.
Can you please help me out?

**Comment by itamarst** [comment, marked as answer]
Fil only supports Linux and macOS at the moment; there is no Windows support. You can use it in Docker or WSL as a workaround.

**Reply by RayenHarchay** [reply]
Thank you for letting me know. Have a great day!

**Reply by tigerinus** [reply]
> Fil only supports Linux and macOS at the moment; there is no Windows support. You can use it in Docker or WSL as a workaround.

I am on Alpine Linux and it's hitting the same issue.

```bash
bash-5.1# uname -a
Linux ml-round-trip-price-prepare-data-service-5bfc5c74cb-2pjd8 5.4.188-104.359.amzn2.x86_64 #1 SMP Thu Apr 14 20:53:13 UTC 2022 x86_64 Linux

bash-5.1# cat /etc/os-release
NAME="Alpine Linux"
ID=alpine
VERSION_ID=3.15.0
PRETTY_NAME="Alpine Linux v3.15"
HOME_URL="https://alpinelinux.org/"
BUG_REPORT_URL="https://bugs.alpinelinux.org/"

bash-5.1# pip install filprofiler
ERROR: Could not find a version that satisfies the requirement filprofiler (from versions: none)
ERROR: No matching distribution found for filprofiler
```

**Reply by itamarst** [reply]
Alpine Linux is not currently supported (and in general I recommend against using it: https://pythonspeed.com/articles/alpine-docker-python/)

**Reply by itamarst** [reply]
It's possible memray works on Alpine (https://bloomberg.github.io/memray/), but it doesn't provide wheels that work on Alpine so you'd have to compile it from scratch and try.

**Comment by willsheppard** [comment]
I'm getting the same problem on Ubuntu 22.04. I'm not using WSL or any other container.

**Reply by itamarst** [reply]
Which version Python? There's no support for 3.12 yet, I haven't had time.
